src/components/modal/v2/styles/components/_header.scss

```
@@ -326,5 +357,54 @@
                 white-space: nowrap;
             }   
         }
+
+        .preapproved {
+            display: flex;
+            align-items: center;
+            flex-direction: row;
+            
+            .preapproved {
+                font-size: 14px;
+                font-weight: 400;
+                line-height: 20px;
+                background-color: #d1e5f9;
+                border-radius: 6px;
+                padding: 2px 8px;
+                margin: 0px 0px 10px 10px;
+
+                @include mobile {
+                    margin: 10px 0px 0px 10px;
+                }
+            }
+        }
+
+        &.checkout {
+            background-color: $ql-gray;
+
+            // logo icons
+            svg {
+                position: relative;
+                bottom: 0px;
+                right: -62px;
+
+                @include mobile {
+                    right: -22px;
+                }
+            }
```

```suggestion
            }
            
            h2 {
                margin-top: 20px;
                margin-bottom: 0px;
            }

```
Let's add a small tweak to the h2 as well for spacing.